### ISSUE TITLE
Post?

### ISSUE THREAD

**Issue by tuzemec** [sheebz/phantom-proxy#19]
Is it possible to emulate the phantomJS post feature as in this example: https://github.com/ariya/phantomjs/blob/master/examples/post.js

```
page.open(someurl, "post", postData, callback);
```

**Comment by rheingolden** [comment]
I agree it would be very useful to be able to http POST to a page in the way shown in the example cited above. Is there any other way of achieving that in the current version?
